crates/bevy_ui/Cargo.toml

```diff
@@ -45,6 +45,7 @@ derive_more = { version = "1", default-features = false, features = [
 nonmax = "0.5"
 smallvec = "1.11"
 accesskit = "0.17"
+tracing = { version = "0.1", default-features = false, features = ["std"] }
```

Comment:
Is it generally expected for crates to use the dep directly instead of using through `bevy_utils`? I was going for the  path of least resistance by using the already dependent on `bevy_utils`

Reply:
nvm i see the main branch has that import..

Reply:
https://github.com/bevyengine/bevy/pull/17161 removed `tracing` from bevy_utils, exporting it from bevy_log; however, that same PR also made any crates that needed to use tracing, use it directly.